File: src/minecraft/monnef/jaffas/trees/ItemJaffaBerryEatable.java
package monnef.jaffas.trees;

import monnef.jaffas.food.item.ItemJaffaFood;

public class ItemJaffaBerryEatable extends ItemJaffaFood {
    public ItemJaffaBerryEatable(int id) {
        super(id);
        setCreativeTab(mod_jaffas_trees.CreativeTab);
    }

    public String getTextureFile() {
        return mod_jaffas_trees.textureFile;
    }
}

File: src/main/scala/monnef/jaffas/trees/item/ItemJaffaBerryEatable.scala
package monnef.jaffas.trees.item

import monnef.jaffas.food.item.ItemJaffaFood
import monnef.jaffas.trees.JaffasTrees
import monnef.jaffas.trees.common.Reference


class ItemJaffaBerryEatable extends ItemJaffaFood {
  setCreativeTab(JaffasTrees.instance.creativeTab)
  setupFromOldDefaultProperties()

  override def getDefaultModName(): String = Reference.ModName

  override def getDefaultSheetNumber(): Int = 2
}
